src/voucherify/voucherify-connector.service.ts

```diff
@@ -95,23 +96,25 @@ export class VoucherifyConnectorService {
   }
 
   async redeemStackableVouchers(
-    coupons: string[],
+    coupons: any,
```

Comment:
I suppose there could be coupons: Coupon[]. I checked this and no errors so far.